### PR TITLE
integration: add rootless test

### LINKED ISSUE #3845: Add test to check rootless containers
Add test to check rootless containers, see https://github.com/kata-containers/kata-containers/pull/2432

cc @fengwang666

### PR DESCRIPTION
**Pull request by devimc**
Add test to check rootles containers

Depends-on: github.com/kata-containers/kata-containers#2432

fixes #3845

cc @fengwang666

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

File: .ci/ci_job_flags.sh [hunk 1]
```diff
@@ -185,6 +185,14 @@ case "${CI_JOB}" in
...
 	export KUBERNETES="yes"
 	;;
+"ROOTLESS")
+	init_ci_flags
+	export CRI_CONTAINERD="no"
+	export CRI_RUNTIME="crio"
+	export CRIO="yes"
+	export KATA_HYPERVISOR="qemu"
+	export KUBERNETES="yes"
+;;
```
Comment (GabyCT) [inline .ci/ci_job_flags.sh:195]: you are missing the alignment with `;;`

**Review by fengwang666 — APPROVED** [review]

**Comment by devimc** [comment]
@GabyCT this doesn't need back port, rootless is a new feature that will land only in `main`

**Review by jodh-intel — APPROVED** [review]
Thanks @devimc.

lgtm

**Comment by devimc** [comment]
@fengwang666  thanks

**Review by GabyCT — APPROVED** [review]

**Pull request merged**